src/Kanbas/Courses/Quizzes/Preview/Question/Single/index.tsx

```diff
@@ -0,0 +1,14 @@
+import RightSide from "../../../../../layout/Content/NotLeftSide/RightSide";
+
+function QuizQuestionPreviewSingle() {
+    return (
+        <>
+            <div></div>
+            <RightSide>
```

Comment:
This is not where RightSide is supposed to be used.
It is supposed to be used at the same level as MiddleContent.
Please refer the home page where all 3 : LeftSide, MiddleContent, and RightSide are used together.